File: Entidad/Persona.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel;

namespace Entidad
{
    public class Persona
    {
        [Key,DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public string Identificacion { get; set;}
        public string Nombre { get; set;}
        public int Edad { get; set; }
        public string Sexo { get; set; }
        public decimal Pulsacion { get; set; }
        public void CalcularPulsaciones()
        {
            if(Sexo.Equals("Femenino")){
                Pulsacion=(220-Edad)/10;
            }else
            {
                Pulsacion=(210-Edad)/10;
            }
        }
    }
}

File: pulsacionesdotnet/ClientApp/src/app/pulsacion/models/persona.ts
import { ThrowStmt } from "@angular/compiler";

export class Persona {
    identificacion:string;
    nombre:string;
    sexo:string;
    edad:number;
    pulsacion:number;

    calcularpul(){
        if(this.sexo=="femenino"){
            this.pulsacion=(220-this.edad)/10; 
        }else{
            this.pulsacion=(210-this.edad)/10;
        }
    }
}
